### DISCUSSION TITLE
Missing button to view list of forks

### DISCUSSION THREAD

**Discussion by Zeex** [community/community#66840] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

There used to be a menu item in the Fork button's dropdown (or was it the button itself?) that allowed me to see all forks of my repo. It's not there anymore - I don't like this change. I know I can click on Insights -> Forks, but the old way was more convenient IMO.

**Comment by aregtech** [comment, marked as answer]
Hi @Zeex ,

To see all forks you should setup filters:
_Step 1:_ Go to *Fork*
_Step 2:_ Click on menu *Repository Types* 
_Step 3:_ Select *Active* and *Inactive* options in the menu as it is shown in this screenshot.
![image](https://github.com/community/community/assets/49253642/c58ccda1-7dfe-4d58-b4df-dc675c89e905)

Then you'll see more repositories. As you see, there are more options. It makes sense to play with them. But to see all forked repositories, in most cases it is enough to select these 2 options.

Once you know how to handle filters, probably you'll like new feature more, because it gives more information. For example, you can display forks sorted with option *Recently updated*, as it is shown in this screenshot. Then you'll have better view whether the repo is just forked or it is actively used by others. In this screenshot, the users have codes not later than 2 weeks. Some of them regularly update, the others just recently forked and have new version:
![image](https://github.com/community/community/assets/49253642/1a683fdb-3ca4-4324-bb53-1a177e2353de)

I hope this comment was helpful 🙂

**Reply by Zeex** [reply]
Thanks.
I think it would make sense to display a link to this page or a quick list of forks for the repo owner when he/she clicks on "Forks" dropdown instead of "You don't have any forks of this repository" message because it's kind of pointless to create a fork of your own repo (or not?).

**Reply by aregtech** [reply]
Agree. It took me also a time until I found how to see the forks. And you still can click on _Save Defaults_.
